lib/experimental/new_embed.dart

```diff
@@ -205,15 +214,17 @@ class ConsoleTabView extends TabView {
   }
 }
 
-class TestTabView extends TabView {
-  final TextAreaElement testEditor;
-
-  const TestTabView(DElement element, this.testEditor) : super(element);
+class TestTabView extends EditorTabView {
+  TestTabView(DElement element, EditorFactory editorFactory)
+      : super(element, editorFactory) {
+        // Tests probably shouldn't change...
+        _editor.readOnly = true;
+      }
 
-  String get testMethod => testEditor.value;
+  String get testMethod => content;
```

Comment:
I'd be fine with just using `content` to get the test code in and out. Your call, though.

Reply:
Done